lib/entities/role.ts

```diff
@@ -12,13 +12,15 @@ export type RoleProps = {
    * Permissions for application sections
    */
   permissions: {
-    ContentDelivery: string
+    ContentDelivery: string[] | string
     ContentModel: string[]
-    Settings: any[]
+    EnvironmentAliases: string[] | string
+    Environment: string
```

Comment:
Looks like it should be plural - Environment**s**

Also, by looking at the example all of them should be `string[] | string`